<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainMenu</class>
 <widget class="QWidget" name="MainMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>736</width>
    <height>508</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QTabWidget" name="tabWidget">
       <property name="tabShape">
        <enum>QTabWidget::Rounded</enum>
       </property>
       <property name="currentIndex">
        <number>3</number>
       </property>
       <property name="usesScrollButtons">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="tabCompose">
        <attribute name="title">
         <string>Compose</string>
        </attribute>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QLabel" name="From">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="baseSize">
                <size>
                 <width>0</width>
                 <height>6</height>
                </size>
               </property>
               <property name="text">
                <string>From</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="compose_userEmail">
               <property name="text">
                <string>TextLabel</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>To</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEdit_compose_to"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>Subject</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEdit_compose_subject"/>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QTextEdit" name="textEdit_compose_emailBody">
             <property name="frameShadow">
              <enum>QFrame::Plain</enum>
             </property>
             <property name="lineWidth">
              <number>3</number>
             </property>
             <property name="horizontalScrollBarPolicy">
              <enum>Qt::ScrollBarAlwaysOff</enum>
             </property>
             <property name="placeholderText">
              <string>Write Email Here....</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_compose_sendMail">
             <property name="text">
              <string>Send Mail</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tabInbox">
        <attribute name="title">
         <string>Inbox</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <widget class="QTableWidget" name="tableWidget_inbox">
           <property name="showGrid">
            <bool>true</bool>
           </property>
           <property name="gridStyle">
            <enum>Qt::DashLine</enum>
           </property>
           <property name="cornerButtonEnabled">
            <bool>true</bool>
           </property>
           <property name="rowCount">
            <number>0</number>
           </property>
           <property name="columnCount">
            <number>4</number>
           </property>
           <attribute name="horizontalHeaderCascadingSectionResizes">
            <bool>true</bool>
           </attribute>
           <attribute name="horizontalHeaderDefaultSectionSize">
            <number>140</number>
           </attribute>
           <attribute name="horizontalHeaderStretchLastSection">
            <bool>true</bool>
           </attribute>
           <column>
            <property name="text">
             <string>From</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Read</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Time</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Subject</string>
            </property>
            <property name="font">
             <font>
              <fontweight>DemiBold</fontweight>
             </font>
            </property>
           </column>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tabSent">
        <attribute name="title">
         <string>Sent Mails</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QTableWidget" name="tableWidget_sentMail">
           <property name="gridStyle">
            <enum>Qt::DashLine</enum>
           </property>
           <attribute name="horizontalHeaderDefaultSectionSize">
            <number>160</number>
           </attribute>
           <attribute name="horizontalHeaderStretchLastSection">
            <bool>true</bool>
           </attribute>
           <column>
            <property name="text">
             <string>To</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Time</string>
            </property>
            <property name="font">
             <font>
              <fontweight>Medium</fontweight>
             </font>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Subject</string>
            </property>
            <property name="font">
             <font>
              <fontweight>DemiBold</fontweight>
             </font>
            </property>
           </column>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tabProfile">
        <attribute name="title">
         <string>Profile</string>
        </attribute>
        <widget class="QPushButton" name="logout">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>15</y>
           <width>83</width>
           <height>29</height>
          </rect>
         </property>
         <property name="text">
          <string>LogOut</string>
         </property>
        </widget>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
